### PR TITLE
feat: re-structure confirmation pages

### PR DESCRIPTION
**Pull request by segun**
## **Description**
Currently, each team works on/owns dozens of different folders and files within our multiple repos, making it harder to:
Define code owners
Get metrics by teams

For proposed solution [see this document](https://docs.google.com/document/d/1dzgWc1LCkYz2lZZ6X8SOaj23iaGYVQ2zJ1FVpOxsU-o/edit)

## **Related issues**

Fixes: [#1952](https://github.com/MetaMask/MetaMask-planning/issues/1952)

## **Screen Recording**

https://github.com/MetaMask/metamask-mobile/assets/44811/34ab76aa-990e-47a5-ac28-191628311e77

## **Pre-merge author checklist**

- [x] I’ve followed [MetaMask Coding Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [x] I've clearly explained what problem this PR is solving and how it is solved.
- [x] I've linked related issues
- [x] I've included manual testing steps
- [x] I've included screenshots/recordings if applicable
- [x] I’ve included tests if applicable
- [x] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [x] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.
- [x] I’ve properly set the pull request status:
  - [ ] In case it's not yet "ready for review", I've set it to "draft".
  - [x] In case it's "ready for review", I've changed it from "draft" to "non-draft".

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

File: ios/Podfile.lock [hunk 1]
```diff
@@ -892,7 +892,7 @@ EXTERNAL SOURCES:
 
 SPEC CHECKSUMS:
   Base64: cecfb41a004124895a7bcee567a89bae5a89d49b
-  boost: 7dcd2de282d72e344012f7d6564d024930a6a440
+  boost: 57d2868c099736d80fcd648bf211b4431e51a558
```
Comment (jpuri) [inline ios/Podfile.lock:895]: 🔪

**Review by blackdevelopa — COMMENTED** [review]
Should we include [transactionSummary](https://github.com/MetaMask/metamask-mobile/blob/cbfdc2a5451194ad4b26cd6b55b5d31e75f1154a/app/components/Views/TransactionSummary/index.js) and [transactionView](https://github.com/MetaMask/metamask-mobile/blob/cbfdc2a5451194ad4b26cd6b55b5d31e75f1154a/app/components/Views/TransactionsView/index.js)

**Review by jpuri — DISMISSED** [review]

**Review by blackdevelopa — DISMISSED** [review]

**Review by jpuri — DISMISSED** [review]

**Review by blackdevelopa — DISMISSED** [review]

**Comment by tommasini** [comment]
Hey @jpuri , it seems sonar cloud is failing for duplications (and not for a lot), could we brace the opportunity and improve the code base a little bit, wdyt?

Also could you had into the PR exploratory testing recordings, doing transactions on the sendflow, and dapps? Just to make sure everything confirmations related is okay

**Comment by segun** [comment]
@jpuri I'm not 100% sure what point 2 is about.

also Is there a way to bypass sonar cloud? as it's stopping me from merging this PR.

**Review by blackdevelopa — DISMISSED** [review]

**Comment by sethkfman** [comment]
@jpuri I will allow the bypass on Sonar when you are ready to merge.

**Review by jpuri — DISMISSED** [review]

**Review by blackdevelopa — DISMISSED** [review]

**Comment by segun** [comment]
This PR have one unresolved conversation that I can't see. [This](https://github.com/orgs/community/discussions/10592#discussioncomment-5672829) seems to the only want to resolve this issue

**Pull request closed without merging**